features/step_definitions/admin_console.rb

```diff
@@ -1,31 +1,22 @@
-# Given /^default admin-console route is stored in the#{OPT_SYM} clipboard$/ do | cb_name |
-#   cb_name ||= :console
-#   cb[cb_name] = route('console', service('console',project('openshift-console', switch: false))).dns(by: admin)
-# end
+Given /^default admin-console route is stored in the#{OPT_SYM} clipboard$/ do | cb_name |
+   cb_name ||= :console
+   cb[cb_name] = route('console', service('console',project('openshift-console', switch: false))).dns(by: admin)
+end
 
 Given /^I open admin console in a browser$/ do
   base_rules = BushSlicer::WebConsoleExecutor::RULES_DIR + "/base/"
   snippets_dir = BushSlicer::WebConsoleExecutor::SNIPPETS_DIR
 
   version = env.webconsole_executor.get_master_version(user, via_rest: true)
-
+  step "default admin-console route is stored in the :console_url clipboard"
   step "I have a browser with:", table(%{
     | rules        | #{base_rules}                            |
     | rules        | lib/rules/web/admin_console/base/        |
     | rules        | lib/rules/web/admin_console/#{version}/  |
-    | base_url     | <%= env.web_console_url %>               |
+    | base_url     | https://<%= cb.console_url %>            |
     | snippets_dir | #{snippets_dir}                          |
   })
-  @result = browser.run_action(:login,
-                               username: user.auth_name,
-                               password: user.password)
-  raise "cannot login to web console" unless @result[:success]
-  @result = browser.run_action(:click_console_selector)
-  raise "cannot click user dropdown menu" unless @result[:success]
-  # we need to open in same tab and I don't know a better way
-  # console_url = browser.browser.a(text: "Cluster Console").href
-  # browser.browser.goto console_url
-  browser.browser.a(text: "Cluster Console").click
+  browser.run_action(:goto_admin_console_root)
```

Comment:
now admin console not accessible like before? How about 3.11 testing?

Reply:
There is no old console on nextGen installed env now, only admin console exists for 4.0. 
With this pr, 3.11 admin console still works well.

Reply:
I want to chat with you about this. Basically the pull request changes way to find admin console on both - 3.11 and 4.0. I preferred the old approach because user will discover admin console through web console. i.e. it resembles actual user more closely. Is this now impossible?

Reply:
@akostadinov , there is no web console deployed in 4.0, user couldn't switch from web console to admin console now, that's why we have to adjust the admin consle login process.

Reply:
Is the plan to drop the web console entirely or it is just a temporary situation? If it is temporary I agree to have this merged. Just I'd request for the old ocde to be commented out so that it is restored once we have a working web console.

Reply:
Another thing is that we have a mechanism in place to select web console by configuration in case it is not discoverable. In case admin console can be used by regular users and we expect to test it with non-admin users, then we need to do the same with admin console. This means:
* add method `admin_console_url` to `environment.rb`
* that method should be similar to existing `web_console_url` method
** will check for configuration option `admin_console_url`
** will use `route('console', service('console',project('openshift-console', switch: false))).dns(by: admin)` when such configuration option is not available
* remove step `default admin-console route is stored in the#{OPT_SYM} clipboard` and call `admin_console_url` method directly where needed

Makes sense?

Reply:
@akostadinov  Make sense to me